otherlibs/win32unix/truncate.c

```diff
@@ -0,0 +1,127 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           */
+/*                                                                        */
+/*   Copyright 2018 Institut National de Recherche en Informatique et     */
+/*     en Automatique.                                                    */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+#define CAML_INTERNALS
+
+#include <sys/types.h>
+#include <caml/mlvalues.h>
+#include <caml/memory.h>
+#include <caml/fail.h>
+#include <caml/signals.h>
+#include <caml/io.h>
+#include <caml/osdeps.h>
+#include "unixsupport.h"
+#include <windows.h>
+
+static int win_truncate_handle(HANDLE fh, long long len)
```

Comment:
```suggestion
static int win_truncate_handle(HANDLE fh, __int64 len)
```
(I'm not certain about the portability of `long long` to the older compilers, so it seems simpler to use the Windows type which encapsulates it - there's `ARCH_INT64_TYPE` defined in `config.h` or just the fact that other files in win32unix.c use `__int64`)

Reply:
Done, switched to `__int64`.